openjdk_regression/playlist.xml

```diff
@@ -14,6 +14,27 @@
 -->
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
 	<include>openjdk_regression.mk</include>
+	<test>
+		<testCaseName>openjdk-custom</testCaseName>
```

Comment:
Better use same name converntions as other testcases: jdk_custom. As we map testcasename or target to build_list.

Reply:
Yes, I was debating on that, or on making a standard naming that would be applied at each of the groups...  to match the jck-custom, openjdk-custom, perf-custom, external-custom, functional-custom... but those other types are difficult to customize, so I will use jdk_custom for now.